### ISSUE TITLE
Text shifts slightly when wrapping to a new line

### ISSUE THREAD

**Issue by Herschel** [onimitch/ffxi-balloon-ashitav4#5]
When Balloon is displaying a long text string, there will be a sudden ~1 pixel shift when the text animation wraps to a new line, probably due to some inconsistency the `gdifonttexture` clip range logic when a newline occurs.

Example video:
https://github.com/onimitch/ffxi-balloon-ashitav4/assets/36278/28677e30-e540-4b51-bdfc-93629d0d1377

Test case:
```
tests.wrap_jitter = {
    'Nebibi',
    "But we ladies-in-waiting mustaru go on being our usual perky-werky selves. I can't let my bed-making standards drop just because times are tough!",
    ''
}
```

**Comment by onimitch** [comment]
I've definitely seen this too. As to when this might get fixed is hard to say since it's a fairly low priority issue. But i'll keep it in mind when I next look at anything relating to the gdifont side.

**Comment by onimitch** [comment]
I think I've found a fix for this, will push my changes later if you want to test it too.

**Comment by onimitch** [comment]
Fixed in [4.2.0](https://github.com/onimitch/ffxi-balloon-ashitav4/releases/tag/4.2.0)
